ci/scripts/image_scripts/share_image.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -ux
+
+export IMAGE_ID
+export PROJECT_ID
+
+OS_SCRIPTS_DIR="$(dirname "$(readlink -f "${0}")")/.."
+
+# shellcheck source=ci/scripts/openstack/infra_defines.sh
+source "${OS_SCRIPTS_DIR}/infra_defines.sh"
```

Comment:
```suggestion
. "${OS_SCRIPTS_DIR}/infra_defines.sh"
```
`source` is deprecated

Reply:
done